learn/async/asynchronous_operations.mdx

```diff
@@ -1,7 +1,7 @@
 ---
 sidebarDepth: 3
 ---
-# Tasks and asynchronous operations
+# Task and asynchronous operations
```

Comment:
Same as my previous comment. Since we're talking about a general type of thing, plural seems more adequate. 

At the very least, it should be consistent: either both plural or both singular, no?

```suggestion
# Tasks and asynchronous operations
```

Reply:
I copied the label from sidebar-learn.json 🤡